enterprise-suite/es-grafana/akka-stream-exporter.json

```diff
@@ -0,0 +1,50 @@
+[
+    {
+        "graphName": "Stream Throughput",
```

Comment:
Should document this is the avg per second

Reply:
That's what the y-axis format documents, although in weird Grafana-ism.  "rps" is reads per sec, "wps" is writes per sec, "ops" is operations per sec. We can put it in the title too if you prefer, just consider this is _everywhere_ so if we change all the rate things to be explained in each title it'll be a little repetitive and inconsistent with titles where we don't explain the unit. If someone wants to see "What's this" they can just open it and see the details.